src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -43,9 +43,7 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
 
         public event EventHandler<ActiveSolutionBindingEventArgs> SolutionBindingChanged;
 
-        public bool IsActiveSolutionBound { get; private set; }
-
-        public string ProjectKey { get; private set; }
+        public BindingConfiguration CurrentBindingConfiguration { get; private set; }
```

Comment:
Naming: _CurrentBindingConfiguration_ is pretty long -  _CurrentConfiguration_ would probably be enough

Reply:
Ok